crates/turbopack-mdx/src/lib.rs

```diff
@@ -58,9 +87,35 @@ async fn into_ecmascript_module_asset(
         anyhow::bail!("Not able to read mdx file content");
     };
 
+    let jsx_runtime = if let Some(runtime) = &*transform_options.jsx_runtime.await? {
+        match runtime.as_str() {
+            "automatic" => Some(mdxjs::JsxRuntime::Automatic),
+            "classic" => Some(mdxjs::JsxRuntime::Classic),
+            _ => None,
+        }
+    } else {
+        None
+    };
+
+    let filepath = if let Some(filepath) = transform_options.filepath {
+        Some(filepath.await?.path.clone())
```

Comment:
Does this need to be a system path? We should use `to_sys_path` instead.

Reply:
it's mostly identifier so not a huge matter afaik.